crates/sui-core/src/tbls/tbls_ids.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use itertools::Itertools;
+use std::collections::HashMap;
+use std::num::NonZeroU32;
+use std::ops::Range;
+use std::unreachable;
+use sui_types::base_types::AuthorityName;
+use sui_types::committee::StakeUnit;
+
+/// Threshold BLS requires unique integer "share IDs".
```

Comment:
Perhaps indicate Threshold BLS is a required component of our upcoming random beacon?